src/scripts/main.js

```diff
@@ -1 +1,22 @@
 'use strict';
+
+window.addEventListener('hashchange', () => {
+  if (window.location.hash === '#menu') {
+    document.body.classList.add('page__body--with-menu');
+  } else {
+    document.body.classList.remove('page__body--with-menu');
+  }
+});
+
+const form = document.querySelector('.form');
+const inputs = document.querySelectorAll('.form__input');
+
+function handler(event) {
+  event.preventDefault();
+
+  inputs.forEach(input => {
+    input.value = '';
+  });
+}
+
+form.addEventListener('submit', handler);
```

Comment:
I transfer data with GET method! and page was reloaded 
![image](https://user-images.githubusercontent.com/105124305/188452006-79f0c2e3-cea1-44d5-88ac-4a65e1bc728a.png)
fix it